community/values/src/main/java/org/neo4j/values/AnyValues.java

```diff
@@ -32,4 +32,23 @@
      */
     public static final Comparator<AnyValue> COMPARATOR =
             new AnyValueComparator( Values.COMPARATOR, VirtualValueGroup::compareTo );
+
+    public static boolean equalityOfValuesInSequences( SequenceValue a, SequenceValue b )
```

Comment:
It looks like this method is responsible for equality check for all possible sequence and it's quite suites as the default implementation of aquals in SequenceValue instead.